packages/cozy-viewer/src/Panel/AI/AIAssistantPanel.jsx

```diff
@@ -49,25 +49,36 @@ const AIAssistantPanel = ({ className }) => {
     }
   }
 
+  const extractFileContent = async (client, fileBlob, file) => {
+    if (isTextFile(file.mime)) {
+      return await fileBlob.text()
```

Comment:
```suggestion
      return fileBlob.text()
```

The await is pointless here